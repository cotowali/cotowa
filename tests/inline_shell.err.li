fn undefined_variable() {
  sh {
    echo %n
  }
}
