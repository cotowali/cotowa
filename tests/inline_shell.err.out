error: inline-shell.err.li 3,10: variable `n` is not defined
    3 |     echo %n
      |          ^^
